core/domain/draft_upgrade_services_test.py

```diff
@@ -158,14 +197,9 @@ def test_convert_states_v32_dict_to_v33_dict(self):
                     }
                 }
             })]
-        expected_draft_change_list = (
-            draft_upgrade_services.DraftUpgradeUtil._convert_states_v32_dict_to_v33_dict(  # pylint: disable=protected-access,line-too-long
-                draft_change_list))
-        self.assertEqual(
-            expected_draft_change_list[1].to_dict(),
-            draft_change_list[1].to_dict())
-        self.assertEqual(
-            expected_draft_change_list[0].to_dict(),
+        # Version 33 adds a showChoicesInShuffledOrder bool, doesn't impact
+        # the second ExplorationChange.
+        draft_change_list_v33 = [
```

Comment:
Add expected_ to the front of any expected variables (that aren't also used for setup).

When doing assertEqual, have "expected" come first, then "actual".